docs/index.md

```diff
@@ -44,48 +53,16 @@ The functions use A1 notation and are compatible with popular spreadsheet softwa
 Excel or Google Sheets, which means that you can easily transfer
 the data and formulas between them.
 
-HyperFormula covers most user-triggered actions such as
-[CRUD operations](/guide/basic-operations.md), [undo/redo](/guide/undo-redo.md),
-and [clipboard operations](/guide/clipboard-operations.md). It also supports
-the use of cross-sheet references, [named expressions](/guide/named-expressions.md),
-[different data types](/guide/types-of-values.md),
-and [custom functions](/guide/custom-functions.md).
+## What can it be used for?
 
-## What can I use it for?
+HyperFormula doesn't assume any existing user interface, making it a great general-purpose library that can be used in various business applications. Here are some examples:
 
-HyperFormula doesn't assume any existing user interface \(UI\),
-which makes it a great general-purpose library that can be used in
-various business applications. Here are some ideas on what you can
-build with HyperFormula:
-
-* Smart documents
-* Educational apps
-* Computation notebooks
-* Data grid components
-* Business logic builders
-* Forms and form builders
-* Spreadsheets
-* Online calculators
-* Low connectivity apps
-
-## Core features
-
-The features that HyperFormula offers make it a flexible and easy
-to use tool for many types of software.
-
-* High-speed formula calculations
-* A library of {{ $page.functionsCount }} built-in functions available in 16 languages
-* Support for cross-sheet references
-* Support for undo/redo
-* Support for copy/cut/paste
-* Support for wildcard characters
-* Support for data sorting
-* Works with popular JavaScript frameworks
-* Written in TypeScript and extensively tested
-* Function syntax compatible with Excel and Google Sheets
-* Double-licensed: GPL/Commercial
-* Actively maintained and supported by the
-[Handsontable](https://handsontable.com/) team
+- Business logic builders
+- Forms and form builders
+- Computation notebooks
+- Smart documents
+- Educational apps
+- Online calculators
```

Comment:
Copied from README.md

Reply:
I discussed with @krzysztofspilka that `- Spreadsheets` should be added here and in the README

Reply:
ok

Reply:
added